vimeo/upload.py

```diff
@@ -21,19 +22,23 @@ class UploadVideoMixin(object):
 
     def upload(self, filename, upgrade_to_1080=False):
         """Upload the named file to Vimeo."""
-        ticket = self.post(self.UPLOAD_ENDPOINT,
-                data={'type': 'streaming',
-                      'upgrade_to_1080': 'true' if upgrade_to_1080 else 'false'})
+        ticket = self.post(
+            self.UPLOAD_ENDPOINT,
+            data={'type': 'streaming',
+                  'upgrade_to_1080': 'true' if upgrade_to_1080 else 'false'},
+            params={'fields': 'upload_link,complete_uri'})
```

Comment:
It'd be preferable if these weren't passed in as defaults. The default library behavior always should be to pull everything back from a resource, and then if you want to slim down your payload, you can supply a list of the only fields you want.

Maybe an just add a new `fields` parameter to these `upload` and `upload_picture` methods, similar to how you did with `upload_texttrack` (just without hardcoding a set of base fields).